packages/transition-frontend/src/config/layers.config.ts

```diff
@@ -4,6 +4,79 @@
  * This file is licensed under the MIT License.
  * License text available at https://opensource.org/licenses/MIT
  */
+
+// Define which layers should be visible for each section
+export const sectionLayers = {
```

Comment:
Do we want the layers configuration file to be aware of the sections? Technically, sections are provided by "modules" (transit offer, transit demand). But for that matter, so are layers probably and that too should not necessarily be a single layer config file... sounds like we'll need to rework our modular approach eventually!

Reply:
We will see later (dixit @tahini)